### PR TITLE
[MODULAR]Combat Cybernetic Implants listing change

### PR DESCRIPTION
**Pull request by SpaceLoveSs13**
## About The Pull Request

1. Militarygrade Combat Cybernetic Implants:- Thermal Eyes, Moth Thermal Eyes removed from medical techfab.
2.  Nightvision cybernetic eyes listed as Utility implant instead of Combat Cybernetic Implant.
3. Night vision cybernetic eyes delisted from Cybernetic Combat Implant technode to Advanced Cybernetic Implant technode.
4. Reviver implant delisted from Advanced Cybernetic Implant technode to Cybernetic Combat Implant technode.

## How This Contributes To The Skyrat Roleplay Experience

For 1, medical techfab should absolutely never have combat implants available in their techfab(or higher). This bypasses the restrictions robotics possess and enforces regularly. Doctors can also otherwise implant themselves and anyone requesting for implants without direct overseeing of robotics or RD, which is required for means of availing combat implants in regulations. (This was slightly updated to reflect changes done on 16/10/2023, only thermal eyes removed as that is definitely something which should be restricted, not the others.)

For 2,3,4 Reviver implant is listed as a combat implant in Contraband policy to prevent confusion among roboticists with its current listing of "Health implant". Same reasoning for Night vision eyes, it has been ruled multiple times that they are treated as just having night vision, not a real combat implant like thermal or xray eyes. Having them taken out of combat listing will help both people seeking it and roboticists ICly without an OOC ruling. (Updated for same reason)

image
From: https://wiki.skyrat13.space/index.php/Contraband

## Proof of Testing

<details>
<summary>Video to show current implant listings in both exosuit fabricator and medical techfab(testing video updated as well)</summary>
 
 
https://github.com/Skyrat-SS13/Skyrat-tg/assets/68121607/da9fc8dd-6316-4220-95a8-141734f1f045

</details>

## Changelog

:cl:SpaceLove
add: Nanotrasen has removed combat restrictions over Nightvision Cybernetic Eyes and it is now readily available in advanced cybernetic implants for all your nightly adventurers!
del: Nanotrasen medical ethics boards has deemed the militarygrade implants such as Thermal eyes implant too risky to keep around in medical techfab, therefore they have been handed over to Robotics only.
/:cl:

### CODE REVIEW

**Comment by YakumoChen** [comment]
The reviver implant really shouldn't be contraband since it doesn't do anything in actual combat, I'll see if I can convince someone to remove it

**Comment by SpaceLoveSs13** [comment]
> The reviver implant really shouldn't be contraband since it doesn't do anything in actual combat, I'll see if I can convince someone to remove it

My pr doesnt make it a contraband. It's only contraband if unauthorized icly. This pr is only to match the in game listing with current policy standing so to help crew distinguish the items more fairly as intended by policy. If the policy gets changed on reviver implant, as I have said in discord, I will change it up.

**Comment by SpaceLoveSs13** [comment]
Hey @GoldenAlpharex , Could I please get a review on this?

**Comment by SpaceLoveSs13** [comment]
PR updated with the suggestions kindly provided by @RatFromTheJungle . PR description, changelog, testing video updated as well to reflect changes.

File: modular_skyrat/modules/implants/code/medical_nodes.dm [hunk 1]
```diff
@@ -16,9 +16,10 @@
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 5000)
 
 /datum/techweb_node/adv_cyber_implants
+	design_ids = list("ci-nv", "ci-nutrimentplus", "ci-surgery", "ci-toolset",)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/implants/code/medical_nodes.dm]:
```suggestion
	design_ids = list("ci-nv", "ci-nutrimentplus", "ci-surgery", "ci-toolset")
```

File: modular_skyrat/modules/implants/code/medical_nodes.dm [hunk 2]
```diff
@@ -16,9 +16,10 @@
...
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 7500)
 
 /datum/techweb_node/combat_cyber_implants
-	design_ids = list("ci-nv", "ci-antidrop", "ci-antistun", "ci-antisleep", "ci-thrusters", "ci-mantis", "ci-flash")
+	design_ids = list("ci-antidrop", "ci-antistun", "ci-antisleep", "ci-thrusters", "ci-mantis", "ci-flash", "ci-reviver",)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/implants/code/medical_nodes.dm]:
```suggestion
	design_ids = list("ci-antidrop", "ci-antistun", "ci-antisleep", "ci-thrusters", "ci-mantis", "ci-flash", "ci-reviver")
```

**Review by GoldenAlpharex — APPROVED** [review]
Should be good to go now!

**Pull request merged**